products/podman-engine.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
+versionCommand: podman --remote version --format '{{.Server.Version}}'
+releaseDateColumn: true
+
+identifiers:
+-   repology: podman
+
+auto:
+  methods:
+  -   git: https://github.com/containers/podman.git
+      regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$
+
+releases:
+-   releaseCycle: "5.1"
+    releaseDate: 2024-05-29
+    eol: false
+    latest: "5.1.2"
+    latestReleaseDate: 2024-07-10
+
+-   releaseCycle: "5.0"
+    releaseDate: 2024-03-19
+    eol: true
+    latest: "5.0.3"
+    latestReleaseDate: 2024-05-10
+
+-   releaseCycle: "4.9"
+    releaseDate: 2024-01-22
+    eol: false
+    latest: "4.9.5"
+    latestReleaseDate: 2024-05-30
+
+-   releaseCycle: "4.8"
+    releaseDate: 2023-11-27
+    eol: true
+    latest: "4.8.3"
+    latestReleaseDate: 2024-01-03
+
+-   releaseCycle: "4.7"
+    releaseDate: 2023-09-27
+    eol: true
+    latest: "4.7.2"
+    latestReleaseDate: 2023-10-31
+
+---
+
+> [Podman](https://podman.io/) is a daemonless, open source, Linux native
+> tool designed to make it easy to find, run, build, share and deploy
+> applications using Open Container Initiative ([OCI](https://www.opencontainers.org/))
+> Containers and Container Images.
+
+Podman Engine is supported by the [Containers Community](https://github.com/containers),
+and as such - there's no list of supported releases.
+This page uses the [Fedora packages](https://packages.fedoraproject.org/pkgs/podman/podman/) as a guide for which releases
```

Comment:
This from remarks from the upstream developers, since they use fedora for their binary distribution.

https://github.com/containers/podman/discussions/23090#discussioncomment-9869667

It is only getting security updates, but then again only for the actual fedora configuration / packaging.

If going by the strict "We only support the latest release upstream generally.", then all the 4.x are EOL

Reply:
Note that both 3.4 and 4.0 and also 4.9 and 5.0 were seeing parallel releases, during a transition time.

Reply:
One could mark **all** but the latest as EOL, and add a comment to look to Fedora and Debian for support?

Including the downstream RHEL and Ubuntu, that is. Maybe add those two explicitly as well, to the comment.

Reply:
Marked them all (but lthe latest release) as "EOL"

Even if fedora/rhel still have upstream branches.

`origin/v4.9`

`origin/v4.9-rhel`